src/opnsense/service/templates/OPNsense/IPFW/ipfw.conf

```diff
@@ -53,7 +53,8 @@ pipe {{ pipe.number }} config bw {{ pipe.bandwidth }}{{ pipe.bandwidthMetric }}/
          if pipe.fqcodel_limit|default('') != '' %} limit {{pipe.fqcodel_limit}} {% endif %} {%
          if pipe.fqcodel_flows|default('') != '' %} flows {{pipe.fqcodel_flows}} {% endif %}
 {% endif %}
-{% endif %}
+{% endif %}{%
```

Comment:
The model doesn’t allow it but elseif would be more robust here as it would indicate what the model actually does.